src/cli/README_SRP_CLIENT.md

```diff
@@ -127,6 +129,31 @@ Removed service list:
     instance:"ins1", name:"_test1._udp", state:Removed, port:777, priority:0, weight:0
 ```
 
+### coder
+
+Usage `srp client coder [enable|disable]`
+
+Enable/Disable use of SRP coder by SRP client.
```

Comment:
Same questions as for the API functions here: what does "enabled" mean?

Reply:
Same as above. Added a note here to indicate it is, as of now, intended for testing.